low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -87,6 +95,16 @@ def get_health_info(self, rpath):
         fru_found = False
         nodes = rpath.strip().split(">")
         leaf_node, _ = self.get_node_details(nodes[-1])
+        try:
+            info["name"] = BuildInfo().get_attribute("NAME")
+            info["version"] = BuildInfo().get_attribute("VERSION")
+            info["build"] = BuildInfo().get_attribute("BUILD")
```

Comment:
You can use single BuildInfo object instead of creating 3 objects.

Reply:
I tried using it earlier, the init function in BuildInfo was getting called each time I was using the object which was initializing conf in "cortx_build_info", multiple times producing following conf error. 
`
cortx.utils.conf_store.error.ConfError: error(22): conf index build_info already exists
`